ios/RCTFBSDK/login/RCTConvert+FBSDKLogin.m

```diff
@@ -21,11 +21,8 @@
 @implementation RCTConvert(FBSDKLogin)
 
 RCT_ENUM_CONVERTER(FBSDKLoginBehavior, (@{
```

Comment:
I think we should just remove login behavior completely since it only has one value now.

Reply:
Actually nevermind, let's keep the prop since it is still used for Android